pkg/operator/ceph/csi/csi.go

```diff
@@ -88,6 +90,11 @@ func (r *ReconcileCSI) setParams(ver *version.Info) error {
 		return errors.Wrap(err, "failed to parse value for 'CSI_ENABLE_HOST_NETWORK'")
 	}
 
+	if CSIParam.EnableReadAffinity, err = strconv.ParseBool(k8sutil.GetValue(r.opConfig.Parameters, "CSI_ENABLE_READ_AFFINITY", "false")); err != nil {
+		return errors.Wrap(err, "failed to parse value for 'CSI_ENABLE_READ_AFFINITY'")
+	}
+	CSIParam.CrushLocationLabels = k8sutil.GetValue(r.opConfig.Parameters, "CSI_CRUSH_LOCATION_LABELS", topology.GetDefaultTopologyLabels())
```

Comment:
I'd like to not expose these in the configmap. Can we just set them directly to the CSI params?

Reply:
> I'd like to not expose these in the configmap. Can we just set them directly to the CSI params?

Yes,
but some users may want flexibility in choosing other non-standard labels to be used here. Maybe it is an external ceph cluster and kubernetes standard topology labels may not be the correct ones to use.